pkg/agent/controller/egress/egress_controller_test.go

```diff
@@ -86,6 +86,43 @@ func (g *antreaClientGetter) GetAntreaClient() (versioned.Interface, error) {
 	return g.clientset, nil
 }
 
+type singleNodeCluster struct {
```

Comment:
nit: maybe call it `fakeSingleNodeCluster`. I think we tend to use a `fake` prefix in the rest of the code for such objects.

Reply:
done